tutorials/clustering.md

```diff
@@ -92,7 +96,12 @@ how many chunks it holds by running the following:
 
 ```sql
 SELECT * FROM timescaledb_information.hypertable;
+```
 
+You can also see the specifics of how a particular hypertable is 
+distributed.
```

Comment:
Suggestion: "You can also see how the data of a distributed hypertable is distributed on the data nodes by using the `hypertable_data_node_relation_size` function."

Reply:
done